### PR TITLE
Ratings Breakdown & Basic .css

### PR DESCRIPTION
**Pull request by RobStafford**
This commit adds the beginning of the Ratings Breakdown functionality as well as basic .css styling to make sure it sits to the left of the ratings list, and not on top of it. Mostly modified my own components, but I also modified api calls & the server.

### CODE REVIEW

File: client/src/components/Ratings/components/IndividualReview.jsx [hunk 1]
```diff
@@ -15,15 +15,13 @@ const IndividualReview = (props) => {
...
-      float: 'left',
       margin: '5px'
     };
 
     return (
-      <div data-testid="individual-review" className="individual-review">
+      <div data-testid="individual-review" className="individual_review">
         <li style={reviewStyle}>
-        <StarRating star_rating={star_rating}></StarRating><span>{moment(date).format('LL')} <br></br> <b>{summary}</b> <br></br> {body} <br></br>...should include pictures<br></br> {reviewer_name} <br></br> {response} <br></br> {recommend ? "I recommend this product!": null}</span>
+        <span><StarRating star_rating={star_rating}></StarRating> {reviewer_name}, {moment(date).format('LL')} <br></br> <b>{summary}</b> <br></br> {body} <br></br>...should include pictures<br></br> {response} <br></br> {recommend ? <img src={'images/transparent-background-check-mark.png'} alt='loading' width="10" height="10"/>: null}{recommend ? " I recommend this product.": null}</span>
```
Comment (DominicMonares) [inline client/src/components/Ratings/components/IndividualReview.jsx]: Line should be split

File: client/src/components/Ratings/components/RatingsList.jsx [hunk 2]
```diff
@@ -9,12 +9,12 @@ const RatingsList = (props) => {
 
   let ratings = props.ratings.map((rating) => {
     return (
-      <IndividualReview key={rating.review_id} star_rating={rating.rating} summary={rating.summary} date={rating.date} body={rating.body} recommend={rating.recommend} reviewer_name={rating.reviewer_name} response={rating.response} helpfulness={rating.helpfulness}></IndividualReview>
+        <IndividualReview key={rating.review_id} star_rating={rating.rating} summary={rating.summary} date={rating.date} body={rating.body} recommend={rating.recommend} reviewer_name={rating.reviewer_name} response={rating.response} helpfulness={rating.helpfulness}></IndividualReview>
```
Comment (DominicMonares) [inline client/src/components/Ratings/components/RatingsList.jsx:12]: Line should be split

File: client/src/components/Ratings/components/StarRating.jsx [hunk 3]
```diff
@@ -5,18 +5,16 @@ const StarRating = (props) => {
   let stars = [];
   for (let i = 0; i < 5; i++) {
     if (i < props.star_rating) {
-      stars.push(<svg key={i} className="solidstar" height="30" width="30"> <polygon points="13, 0, 18, 10, 28, 11, 21, 18, 24, 28, 13, 23, 4, 28, 7, 18, 0, 11, 10, 10, 13, 0" fill="black" /></svg>);
+      stars.push(<svg key={i} className="solidstar" height="15" width="15"> <polygon points="6.5, 0, 9, 5, 14, 5.5, 10.5, 9, 12, 14, 6.5, 11.5, 2, 14, 3.5, 9, 0, 5.5, 5, 5, 6.5, 0" fill="black" /></svg>);
     } else {
-      stars.push(<svg key={i} className="hollowstar" height="30" width="30"> <polygon points="13, 0, 18, 10, 28, 11, 21, 18, 24, 28, 13, 23, 4, 28, 7, 18, 0, 11, 10, 10, 13, 0" fill="white" stroke="black" /></svg>);
+      stars.push(<svg key={i} className="hollowstar" height="15" width="15"> <polygon points="6.5, 0, 9, 5, 14, 5.5, 10.5, 9, 12, 14, 6.5, 11.5, 2, 14, 3.5, 9, 0, 5.5, 5, 5, 6.5, 0" fill="white" stroke="black" /></svg>);
```
Comment (DominicMonares) [inline client/src/components/Ratings/components/StarRating.jsx:10]: Might want to move points arrays to a variable

**Review by DominicMonares — CHANGES_REQUESTED** [review]
Looks great overall! Only things I notice are lines that could be split in order to avoid super long lines (line 24 in IndividualReview for example). I tend to split lines around the 110-120 character mark, I can't remember what exact length they recommended in the past but I want to say it's between 120 and 160. It doesn't necessarily have to meet that exact range, but I would avoid exceeding 180 characters at the most. Other than that, it's an approval from me.

File: client/src/components/Ratings/components/IndividualReview.jsx (hunk 1 above)
Comment (RobStafford) [reply client/src/components/Ratings/components/IndividualReview.jsx]: Hi Dom--I broke it up!

**Pull request merged**